packages/components/src/VirtualizedList/utils/tablerow.js

```diff
@@ -26,8 +26,13 @@ export function insertSelectionConfiguration({ children, isSelected, selectionTo
 				cellDataGetter={({ rowData }) => isSelected(rowData)}
 				columnData={{ label: 'Select this element', onChange: selectionToggle }}
 				{...CellCheckboxRenderer}
-			/>);
+			/>
+		);
 		contentsConfiguration = [toggleColumn].concat(contentsConfiguration);
+	} else if (selectionToggle || isSelected) {
+		console.warn(
```

Comment:
I suspect this to be printed on each render. If this is the case -> avoid this behavior and add a marker to trigger the print only once.

Reply:
I don't know : 
* the check should be here (in this function) because it manages the selection feature
* it is not a component but a function. This means that the flag will be managed by the instance of List. This breaks the 1st point
* having it on each render, is not a problem because it is a dev issue. But in prod mode, yes it is a problem

So what I can do is not print it if not in production mode